docs/contributor-docs/release-procedure.md

```diff
@@ -0,0 +1,88 @@
+# Release Procedure
+
+This document is a Core team guide, provided publicly to ensure transparancy in the release process.
+
+## Versioning
+
+Jellyfin uses [semantic versioning](https://semver.org). All releases will have versions in the `X.Y.Z` format, starting from `10.0.0`. Note however that the `10.Y.Z` release chain represents the "cleanup" of the codebase, so it should be accepted that `10.Y.Z` breaks all compatibility, at some point, with previous Emby-compatible interfaces. Our versioning will typically follow the patterns below:
+
+#### `X` - Major breaking versions
+
+* Breaks compatibility with the HTTP and/or plugin APIs.
```

Comment:
I mention this specifically in the paragraph above:

>Note however that the `10.Y.Z` release chain represents the "cleanup" of the codebase, so it should be accepted that `10.Y.Z` breaks all compatibility, at some point, with previous Emby-compatible interfaces.

I think this covers the intention, since we weren't actually building plugins properly or officially before `10.2.0` so them being re-broken is an exception, but I added this explicitly:

>and may also break compatibility with previous `10.Y` releases if required for later cleanup work.